docs/src/pages/[platform]/connected-components/authenticator/headless/headless-usage.web.mdx

```diff
@@ -38,33 +38,101 @@ Here's an example that conditionally renders different components based on curre
+          </TableRow>
+          <TableRow>
+            <TableCell>`isPending`</TableCell>
+            <TableCell>Whether service API call is in progress</TableCell>
+            <TableCell>`boolean`</TableCell>
+          </TableRow>
+          <TableRow>
+            <TableCell>`codeDeliveryDetail`</TableCell>
+            <TableCell>Provides detail on where confirm sign up code is sent to.</TableCell>
+            <TableCell>[`CodeDeliveryDetail`](https://github.com/aws-amplify/amplify-js/blob/6c4d4b5fc737652f88dcdfa26bdb02a1defdfa2e/packages/amazon-cognito-identity-js/index.d.ts#L9-L13)</TableCell>
+          </TableRow>
+        </TableBody>
+      </Table>
+    </View>
   </TabItem>
   <TabItem title="Trigger Transition">
 
 These helper functions trigger transition to another `route`. Note that any invalid transition (e.g. `sign-in` to `authenticated` directly) will be no-op.
 
-| Name                | Description                                                                                                                                                                                                                             | Type                         |
-| ------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------- |
-| `toSignIn`          | Transitions to `signIn`. Allowed from `signUp`, `confirmSignUp`, `confirmSignIn`, `setupTOTP`, `resetPassword`, and `confirmResetPassword`                                                                                              | `() => void`                 |
-| `toSignUp`          | Transitions to `signUp`. Allowed from `signIn`.                                                                                                                                                                                         | `() => void`                 |
-| `toResetPassword`   | Transitions to `resetPassword`. Allowed from `signIn`.                                                                                                                                                                                  | `() => void`                 |
-| `toFederatedSignIn` | Transitions to `provider`'s federated sign in page. Supported `provider` values can be found [here](https://github.com/aws-amplify/amplify-ui/blob/96830f6a34a417aa9bc6329c839679bd10da84f0/packages/ui/src/helpers/auth.ts#L104-L109). | `(provider: string) => void` |
-| `skipVerification`  | Skips verification process. Allowed from `verifyUser` and `confirmVerifyUser`                                                                                                                                                           | `() => void`                 |
-
+    <View overflow="scroll">
```

Comment:
As followup, can we move this into a tsx file so we get intellisense and type errors?